conf/ocsci/vsphere_scale_cluster_rhos.yaml

```diff
@@ -0,0 +1,16 @@
+---
+# This config is suppose to work on most of DCs we have.
+DEPLOYMENT:
+  allow_lower_instance_requirements: false
+ENV_DATA:
+  scale_up: True
+  nodes_scaleup_count: '3'
+  platform: 'vsphere'
+  deployment_type: 'upi'
+  worker_replicas: 3
+  master_replicas: 3
+  worker_num_cpus: '16'
+  master_num_cpus: '4'
+  master_memory: '16384'
+  compute_memory: '65536'
```

Comment:
can we reduce this to 48GB?

Reply:
Yes sure. Do you think it will be better here?